src/main/proto/ga4gh/genotype_phenotype_service.proto

```diff
@@ -80,28 +89,34 @@ message EvidenceQuery {
     // ECO or OBI is recommended
     OntologyTerm evidenceType = 1;
 
-    string description = 2;  /*regex*/
+    // regex
+    string description = 2;
 
-    repeated ExternalIdentifier external_identifiers = 3; /* new field */
+    // new field
+    repeated ExternalIdentifier external_identifiers = 3;
 }
 
 message SearchPhenotypesRequest {
     // The `PhenotypeAssociationSet` to search. Mandatory
     string phenotype_association_set_id = 1;
 
+    // Phenotype ID
     string id = 2;
 
-    string description = 3; /* The system should support regular expression matches */
+    // The system should support regular expression matches
+    string description = 3;
 
+    // type
```

Comment:
Details on how the request fields are to be handled should be in this document. I added boilerplate when there was not other description.